### PR TITLE
{lang}[gfbf/2023a] R v4.3.1

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso
EB_ARGS="--skip"

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Comment by fizwit** [comment]
Would consider adding ggplot2?  ggplot2 is a dependency 67 times in R-4.3.0.

**Comment by verdurin** [comment]
> Would consider adding ggplot2? ggplot2 is a dependency 67 times in R-4.3.0.

What does it require?

**Comment by fizwit** [comment]
hmmm, ggplot2 requires 14 dependencies that are not in the current R-4.3.1.  I have put ggplot2 into a new easyconfig [R-Tidyverse-4.3.1](https://github.com/FredHutch/easybuild-life-sciences/blob/main/fh_easyconfigs/r/R-Tidyverse/R-Tidyverse-4.3.1-gfbf-2022b.eb) which is larger than the base R.  Started with eight libraries with R-Tidyverse and processed with easy_update. The result is about 130 libraries.

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
On Wed, Jul 05, 2023 at 02:55:44AM -0700, Kenneth Hoste wrote:
> @boegel requested changes on this pull request.
> 
> 
> 
> > @@ -0,0 +1,374 @@
> +name = 'R'
> +version = '4.3.1'
> +
> +homepage = 'https://www.r-project.org/'
> +description = """R is a free software environment for statistical computing
> + and graphics."""
> +
> +toolchain = {'name': 'gfbf', 'version': '2022b'}
> 
> Since we already have an `R` for `2022b`, shouldn't we bump this to `2023a` (once `foss/2023a` is set in stone)?
> 

Maybe. 

We need it now, which is why I went with an existing toolchain.

**Comment by verdurin** [comment]
> hmmm, ggplot2 requires 14 dependencies that are not in the current R-4.3.1. I have put ggplot2 into a new easyconfig [R-Tidyverse-4.3.1](https://github.com/FredHutch/easybuild-life-sciences/blob/main/fh_easyconfigs/r/R-Tidyverse/R-Tidyverse-4.3.1-gfbf-2022b.eb) which is larger than the base R. Started with eight libraries with R-Tidyverse and processed with easy_update. The result is about 130 libraries.

Thanks. Are you happy with that approach then, rather than adding `ggplot2` here?

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/r/R/R-4.3.1-gfbf-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+    ('bzip2', '1.0.8'),
+    ('XZ', '5.4.2'),
+    ('zlib', '1.2.13'),
+    ('SQLite', '3.42.0'),
+    ('PCRE2', '10.42'),
+    ('libpng', '1.6.39'),  # for plotting in R
+    ('libjpeg-turbo', '2.1.5.1'),  # for plottting in R
+    ('LibTIFF', '4.5.0'),
+    ('Java', '11', '', SYSTEM),
+    ('libgit2', '1.7.0'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/r/R/R-4.3.1-gfbf-2023a.eb:35]: A newer `libgit2` was already merged in https://github.com/easybuilders/easybuild-easyconfigs/pull/18606
```suggestion
    ('libgit2', '1.7.1'),
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]
@verdurin I updated to R 4.3.2, added the dependencies and updated the extensions in https://github.com/verdurin/easybuild-easyconfigs/pull/8

**Pull request merged**